src/animals/__test__/animals.service.spec.ts

```diff
@@ -40,5 +126,204 @@ describe('AnimalsService', () => {
       const result = await service.findAll();
       expect(result).toMatchSnapshot();
     });
+
+    it('should throw Internal Exception Error when something goes wrong with mongodb connection', async () => {
+      jest.spyOn(model, 'find').mockImplementation(() => {
+        throw new Error('mongodb down');
+      });
+
+      expect(service.findAll).rejects.toMatchSnapshot();
```

Comment:
tutaj chcialbym abys sprawdzil czy dostajesz error domenowy w tym wypadku to podejscie nie jest takie przjrzyste

Reply:
nie za bardzo rozumiem, co dokładnie mam zrobić??

Reply:
zrobione